x-pack/legacy/plugins/infra/public/pages/metrics/containers/with_metrics_time.tsx

```diff
@@ -42,19 +42,15 @@ export const useMetricsTime = () => {
     interval: '>=1m',
   });
 
-  const parsedFrom = dateMath.parse(timeRange.from);
-  const parsedTo = dateMath.parse(timeRange.to, { roundUp: true });
```

Comment:
:information_source: This always produced a new `Moment` and thereby caused the memoization below to have no effect. It was therefore moved into the memoization function's body so the original value could be used as the memoization key.